app/scripts/lib/experiments/grouping-rules/token-code.js

```diff
@@ -6,17 +6,46 @@
 
 const BaseGroupingRule = require('./base');
 const Constants = require('../../../lib/constants');
-const GROUPS = ['control', 'treatment-code', 'treatment-link'];
+const GROUPS_DEFAULT = ['control', 'treatment-code', 'treatment-link'];
+
 const ROLLOUT_CLIENTS = {
+  '37fdfa37698f251a': {
+    groups: GROUPS_DEFAULT,
+    name: 'Lockbox Extension',
+    rolloutRate: 0.0
+  },
   '3a1f53aabe17ba32': {
+    groups: GROUPS_DEFAULT,
     name: 'Firefox Add-ons',
     rolloutRate: 0.0 // Rollout rate between 0..1
   },
+  '7f368c6886429f19': {
+    groups: ['treatment-code'],
+    name: 'Firefox Notes Android Dev',
+    rolloutRate: 0.0
+  },
+  '98adfa37698f255b': {
+    groups: ['treatment-code'],
+    name: 'Lockbox Extension iOS',
+    rolloutRate: 0.0
+  },
+  'b7d74070a481bc11': {
+    groups: ['treatment-code'],
+    name: 'Firefox Notes',
```

Comment:
@vladikoff I remember we talked about Notes using codes instead of links for sign-in confirmation. I added them in here just incase we wanted to enable it, although I am not sure who makes the final decision for that?